test-page/src/views/Category.vue

```diff
@@ -117,8 +117,12 @@
           <SfProductCard
             v-for="(product, i) in products"
             :key="i"
-            :title="product.name"
-            image="/img/product_thumb.png"
+            :title="product.name || ''"
+            :image="
+              product.media && product.media.length
+                ? product.media[0].media.url
+                : '/img/product_thumb.png'
```

Comment:
Not required right now but maybe we may extract getting a default thumbnail to some method? We will use that in product and cms pages also. :)

Edit:
After a quick look, I am not sure if there is a default thumbnail for products with no cover in Shopware. I guess Shopware paints default one with polygons. I did not find that in admin panel. In the SW6 codebase I found only lines for throwing an exception in case of not image found.

![image](https://user-images.githubusercontent.com/1880231/67331148-b3bee680-f51d-11e9-85c3-2c977804e568.png)

Reply:
yeah, this is only temporary, we need wrapper over this component to take Shopwate product and put it in StorefrontUI item :)
and agree - helpers in this area will be necessary!